Godeps/Godeps.json

```diff
@@ -5,13 +5,13 @@
 	"Deps": [
 		{
 			"ImportPath": "github.com/apache/openwhisk-client-go/whisk",
-			"Comment": "1.0.0-1-gee5b870",
-			"Rev": "ee5b8709787cd37201c42e38040e9709f6d1e9c8"
+			"Comment": "1.1.0-4-g0539c51",
+			"Rev": "0539c51f48f30aedffd6a846653a734cbc60bb0b"
```

Comment:
Hmm the hash I see is 17d556327cd39cb6d6854667fa75c35ef9f2e6f1.

Reply:
Fixed. I added my branch commit hash Earlier. But now I updated this hash with the hash that was created after merging PR into master.